lib/ability.rb

```diff
@@ -86,12 +113,13 @@ def initialize(user, resource, controller)
 
       if user.facility_director_of?(resource)
         can [ :activate, :deactivate ], ExternalService
+        can :disputed, [Order, Reservation]
```

Comment:
You've added this ability, but it looks like you're testing against `:manage`. Should we be using `disputed`?